src/components/Tag/Tag.tsx

```diff
@@ -6,14 +6,16 @@ import styles from "./Tag.module.css";
 export const Tag = ({
   active,
   label,
+  level,
   onRemove,
 }: {
   active?: boolean;
   label: string;
+  level: string;
```

Comment:
I suggest we make this prop optional (and perhaps we can call it title?), think this is why build is failing